homeassistant/components/weheat/strings.json

```diff
@@ -84,6 +84,15 @@
       },
       "electricity_used": {
         "name": "Electricity used"
+      },
+      "energy_output": {
+        "name": "Total energy output"
+      },
+      "compressor_rpm": {
+        "name": "Compressor RPM"
+      },
```

Comment:
Ideally we don't have the unit of measurement in the entity name, is there a better name we can use here?

Reply:
Not really sure. We have a RPM and a percentage field.

Reply:
`Compressor speed` and `Compressor usage`?

Reply:
Alrighty, updated the names.